modules/oadp-storage-class-mapping-oadp.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+[id="oadp-storage-class-mapping-oadp_{context}"]
+:_mod-docs-content-type: PROCEDURE
+
+[id=storage-class-mapping-oadp_{context}]
+= Storage class mapping with OADP
```

Comment:
Proc should have a gerund title, does this work..?
```suggestion
= Mapping storage classes with OADP
```

Reply:
Thank you!!!